CacheManager.StackExchange.Redis.SelfCleaning/SelfCleaningRedisCacheHandle.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using CacheManager.Core;
+using CacheManager.Core.Internal;
+using CacheManager.Core.Logging;
+using CacheManager.Redis;
+using CacheManager.StackExchange.Redis.SelfCleaning.Core;
+using CacheManager.StackExchange.Redis.SelfCleaning.Timers;
+using StackExchange.Redis;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning
+{
+    public class SelfCleaningRedisCacheHandle<TCacheValue> : RedisCacheHandle<TCacheValue>, IStartable
+    {
+        private readonly IDatabase _redisDatabase;
+        private readonly ITimer _cleanupTimer;
+        private readonly TimeSpan _slidingExpiration;
```

Comment:
Dont you think that TimeToLive is better?
If you want to stick with that name because its being used in CacheManager thats fine, just asking

Reply:
Changed